happi/loader.py

```diff
@@ -19,57 +22,80 @@
 main_event_loop = None
 
 
-def fill_template(template, device, enforce_type=False):
+def fill_template(
+    template: str,
+    device: HappiItem,
+    enforce_type: bool = False
+) -> Any:
     """
-    Fill a Jinja2 template using information from a device
+    Fill a Jinja2 template using information from a device.
 
     Parameters
     ----------
     template : str
-        Jinja2 template
+        Jinja2 template source.
 
-    device : happi.Device
-        Any device container
+    device : HappItem
```

Comment:
HappItem > Happ**i**Item ?

I almost like the mispelled version more

Reply:
I unironically think happ is a better name then happi, sounds like some new hip database. Have you stored all your devices in happ?

Reply:
I'm digging it!

... I wish my typos were always so entertaining and thought-provoking 😁